src/test/scala/com/asidatascience/configuration/DynamicConfigurationFromHttpSpec.scala

```diff
@@ -29,6 +29,7 @@ with ScalaFutures {
 
   override def afterAll(): Unit = {
     actorSystem.terminate.futureValue
```

Comment:
I'd rather use

```scala
TestKit.shutdownActorSystem(actorSystem)
```

This already returns Unit and checks the system was terminated successfully, so we don't have to block and explicitly swallow the return type.